packages/builder/src/components/backend/DataTable/modals/CreateEditColumn.svelte

```diff
@@ -287,6 +289,12 @@
     if (saveColumn.type !== LINK_TYPE) {
       delete saveColumn.fieldName
     }
+    if (saveColumn.subtype === USER_TYPE) {
```

Comment:
should you confirm it is a bbReference type first just to make sure that it is a user column?

Reply:
Now checking bb_reference type